### ISSUE TITLE
Accessibility testing template spreadsheet is access-restricted

### ISSUE THREAD

**Issue by thibaudcolas** [alphagov/wcag-primer#84]
On the [document issues wiki page](https://github.com/alphagov/wcag-primer/wiki/document-issues), there is a link to a spreadsheet which isn’t publicly viewable:

> [Richard Morton's WCAG 2.0 accessibility testing template (XLSx)](https://docs.google.com/spreadsheets/d/1cXfCqXVLSnPNd4-2qnfPKtd2W--ki7B2/edit?usp=sharing&ouid=107464948553210397587&rtpof=true&sd=true) - note this is based on an older version of WCAG)

It would be nice if this was publicly viewable – or if this is intentional, maybe add a "(access restricted)" note?

**Comment by andrewhick** [comment]
Hi @thibaudcolas , thanks for flagging. I've just gone into the spreadsheet and the Share button shows 'Anyone on the internet with the link can view'. I've also accessed it from a browser I'm not signed in to Google with.

What do you see when you attempt to access the spreadsheet? Could your network have restricted access to Google Sheets?

**Comment by andrewhick** [comment]
We have recently updated some permissions in the meantime, so please retry and let us know 🙏

**Comment by selfthinker** [comment]
Sorry about that. We had to move some folders around internally and some permissions got lost during that process. That (and two more files) should be public again.

**Comment by thibaudcolas** [comment]
Thank you, I can access now :) Before I was just seeing Google Drive’s default dialog to request access.
